code/game/objects/structures/stool_bed_chair_nest/xeno_nest.dm

```diff
@@ -128,11 +128,7 @@
 	if(LAZYLEN(buckled_mobs))
 		. += image("icon_state" = "nest_overlay", "layer" = LYING_MOB_LAYER + 0.1)
 
-
-/obj/structure/bed/nest/flamer_fire_act(burnlevel)
-	take_damage(burnlevel * 2, BURN, FIRE)
-
-/obj/structure/bed/nest/fire_act()
+/obj/structure/bed/nest/fire_act(burn_level)
 	take_damage(50, BURN, FIRE)
```

Comment:
Cutting the scaling? No idea whether it matters for gameplay, admittedly.

Reply:
done